### PR TITLE
test: Subsystem test csv files inside zip

### PR DESCRIPTION
**Pull request by JesperGomes**
# Description

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are written and executed locally
- [x] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/dotnet/subsystem-tests/Features/SettlementReports/Fixtures/Databricks/FilesApiClient.cs [hunk 1]
```diff
@@ -49,4 +49,25 @@ public async Task<FileInfo> GetFileInfoAsync(string filePath, CancellationToken
...
+
+    /// <summary>
+    /// Gets a stream to the file.
+    /// </summary>
+    /// <param name="filePath">The absolute path of the file.</param>
+    /// <param name="cancellationToken">Cancellation token.</param>
+    /// <returns>A stream to the file.</returns>
+    public async Task<Stream> GetFileStreamAsync(string filePath, CancellationToken cancellationToken = default)
+    {
+        var url = $"{ApiVersion}/fs/files{filePath}";
```
Comment (BjarkeMeier) [inline source/dotnet/subsystem-tests/Features/SettlementReports/Fixtures/Databricks/FilesApiClient.cs:61]: Can we avoid duplicating this URL construction in each method?

File: source/dotnet/subsystem-tests/Features/SettlementReports/SettlementReportBalanceFixingJobGeneratesZipScenario.cs [hunk 2]
```diff
@@ -102,10 +102,31 @@ public async Task AndThen_OutputFileIsGeneratedAtExpectedLocation()
         outputFileInfo.Should().NotBeNull($"because we expected the file (relative path) '{Fixture.ScenarioState.ExpectedRelativeOutputFilePath}' to exists.");
     }
 
+    [ScenarioStep(4)]
```
Comment (BjarkeMeier) [inline source/dotnet/subsystem-tests/Features/SettlementReports/SettlementReportBalanceFixingJobGeneratesZipScenario.cs:105]: Should this be tested on dev_002/dev_003 before merging?

Comment (JesperGomes) [reply source/dotnet/subsystem-tests/Features/SettlementReports/SettlementReportBalanceFixingJobGeneratesZipScenario.cs:105]: I did but forgot the checkbox

**Review by BjarkeMeier — APPROVED** [review]

File: source/dotnet/subsystem-tests/Features/SettlementReports/Fixtures/Databricks/FilesApiClient.cs (hunk 1 above)
Comment (JesperGomes) [reply source/dotnet/subsystem-tests/Features/SettlementReports/Fixtures/Databricks/FilesApiClient.cs]: Done

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**